### ISSUE TITLE
Keyboard input

### ISSUE THREAD

**Issue by Tuqz** [Tuqz/athelia#1]
The 'block' techdemo fails to respond to user input.

**Comment by jacob-hartmann** [comment]
I can confirmed this on Windows 7. I am working on getting debug symbols built in an attempt to track this down.

**Comment by Jacob843** [comment]
Tuqz, I just noticed that both techdemos/block/main.cpp and techdemos/template/main.cpp have main() functions. One calls OIS while one doesn't. On my system, scons seems to be compiling template's main (the log name is "LOG_NAME", which matches up locally), when it should be compiling block's main. This should be fixed. Additionally, note that I have created the nightly branch, where I will be pushing most of my changes.
